### ISSUE TITLE
calling a helpline from Skype gives an error

### ISSUE THREAD

**Issue by alanb2718** [bmlt-enabled/yap#571]
Trying to call a helpline from Skype (using Skype's dial-out facility) gives an error for yap 4.0.0.  This used to work with 3.9.9.  It looks like this occurs because the call from Skype looks like it is from a weird phone number.

To reproduce: call a US yap number from Skype.  It may also be worth trying some other ways of calling a US helpline, for example, calling from a number in Europe.  (If it's just calling from Skype that breaks, this may not be terribly important to fix; but I can imagine wanting to be able to call a helpline from another country.)

Here is the error from the Twilio dashboard, with timestamp 2021-11-30 22:34:36 UTC:

`An attempt to retrieve content from https://bmlt.wszf.org/yap/meeting-search.php?Latitude=47.6062095&Longitude=-122.3320708 returned the HTTP status code 500`

Here is the error from the Laravel log:

```
[2021-11-30 14:34:36] production.ERROR: [HTTP 400] Unable to create record: The 'To' number +266696687 is not a valid phone number. (View: /home/customer/www/bmlt.wszf.org/public_html/yap/resources/views/legacy.blade.php) {"view":{"view":"/home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Version.php","data":[]},"exception":"[object] (Facade\\Ignition\\Exceptions\\ViewException(code: 0): [HTTP 400] Unable to create record: The 'To' number +266696687 is not a valid phone number. (View: /home/customer/www/bmlt.wszf.org/public_html/yap/resources/views/legacy.blade.php) at /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Version.php:85)
[stacktrace]
#0 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Version.php(219): Twilio\\Version->exception(Object(Twilio\\Http\\Response), 'Unable to creat...')
#1 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/MessageList.php(66): Twilio\\Version->create('POST', '/Accounts/AC3af...', Array, Array)
#2 /home/customer/www/bmlt.wszf.org/public_html/yap/legacy/meeting-search.php(50): Twilio\\Rest\\Api\\V2010\\Account\\MessageList->create('+266696687', Object(Twilio\\Values))
#3 /home/customer/www/bmlt.wszf.org/public_html/yap/legacy/meeting-search.php(136): sendSms('Meeting Results...')
#4 /home/customer/www/bmlt.wszf.org/public_html/yap/resources/views/legacy.blade.php(1): require_once('/home/customer/...')
#5 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(107): require('/home/customer/...')
#6 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(108): Illuminate\\Filesystem\\Filesystem::Illuminate\\Filesystem\\{closure}()
#7 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\\Filesystem\\Filesystem->getRequire('/home/customer/...', Array)
#8 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(61): Illuminate\\View\\Engines\\PhpEngine->evaluatePath('/home/customer/...', Array)
#9 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/facade/ignition/src/Views/Engines/CompilerEngine.php(37): Illuminate\\View\\Engines\\CompilerEngine->get('/home/customer/...', Array)
#10 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/View.php(139): Facade\\Ignition\\Views\\Engines\\CompilerEngine->get('/home/customer/...', Array)
#11 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#12 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#13 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Http/Response.php(69): Illuminate\\View\\View->render()
#14 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\\Http\\Response->setContent(Object(Illuminate\\View\\View))
#15 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(55): Illuminate\\Http\\Response->__construct(Object(Illuminate\\View\\View), 200, Array)
#16 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(85): Illuminate\\Routing\\ResponseFactory->make(Object(Illuminate\\View\\View), 200, Array)
#17 /home/customer/www/bmlt.wszf.org/public_html/yap/app/Http/Controllers/LegacyController.php(36): Illuminate\\Routing\\ResponseFactory->view('legacy', Array)
#18 /home/customer/www/bmlt.wszf.org/public_html/yap/app/Http/Controllers/LegacyController.php(18): App\\Http\\Controllers\\LegacyController->renderPath('/home/customer/...', 'text/xml')
#19 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\\Http\\Controllers\\LegacyController->index(Object(Illuminate\\Http\\Request), 'meeting-search....')
#20 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction('index', Array)
#21 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Route.php(261): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(App\\Http\\Controllers\\LegacyController), 'index')
#22 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Route.php(204): Illuminate\\Routing\\Route->runController()
#23 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(695): Illuminate\\Routing\\Route->run()
#24 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#25 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#26 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#27 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#28 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#29 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#30 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#31 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#32 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest(Object(Illuminate\\Http\\Request), Object(Illuminate\\Session\\Store), Object(Closure))
#33 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Session\\Middleware\\StartSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#34 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#35 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#36 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#37 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#38 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#39 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(697): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#40 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(672): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))
#41 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(636): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))
#42 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(625): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))
#43 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(166): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))
#44 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))
#45 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#46 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#47 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#48 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#49 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#50 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#51 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#52 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#53 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#54 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#55 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/fruitcake/laravel-cors/src/HandleCors.php(38): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#56 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fruitcake\\Cors\\HandleCors->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#57 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/fideloper/proxy/src/TrustProxies.php(57): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#58 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fideloper\\Proxy\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#59 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#60 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(141): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#61 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(110): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#62 /home/customer/www/bmlt.wszf.org/public_html/yap/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#63 {main}

[previous exception] [object] (Twilio\\Exceptions\\RestException(code: 21211): [HTTP 400] Unable to create record: The 'To' number +266696687 is not a valid phone number. at /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Version.php:85)
[stacktrace]
#0 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Version.php(219): Twilio\\Version->exception(Object(Twilio\\Http\\Response), 'Unable to creat...')
#1 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/MessageList.php(66): Twilio\\Version->create('POST', '/Accounts/AC3af...', Array, Array)
#2 /home/customer/www/bmlt.wszf.org/public_html/yap/legacy/meeting-search.php(50): Twilio\\Rest\\Api\\V2010\\Account\\MessageList->create('+266696687', Object(Twilio\\Values))
#3 /home/customer/www/bmlt.wszf.org/public_html/yap/legacy/meeting-search.php(136): sendSms('Meeting Results...')
#4 /home/customer/www/bmlt.wszf.org/public_html/yap/storage/framework/views/696b0896bb34761cb4c95ea068c4ad4b0d99f02e.php(1): require_once('/home/customer/...')
#5 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(107): require('/home/customer/...')
#6 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(108): Illuminate\\Filesystem\\Filesystem::Illuminate\\Filesystem\\{closure}()
#7 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\\Filesystem\\Filesystem->getRequire('/home/customer/...', Array)
#8 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(61): Illuminate\\View\\Engines\\PhpEngine->evaluatePath('/home/customer/...', Array)
#9 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/facade/ignition/src/Views/Engines/CompilerEngine.php(37): Illuminate\\View\\Engines\\CompilerEngine->get('/home/customer/...', Array)
#10 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/View.php(139): Facade\\Ignition\\Views\\Engines\\CompilerEngine->get('/home/customer/...', Array)
#11 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#12 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#13 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Http/Response.php(69): Illuminate\\View\\View->render()
#14 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\\Http\\Response->setContent(Object(Illuminate\\View\\View))
#15 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(55): Illuminate\\Http\\Response->__construct(Object(Illuminate\\View\\View), 200, Array)
#16 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(85): Illuminate\\Routing\\ResponseFactory->make(Object(Illuminate\\View\\View), 200, Array)
#17 /home/customer/www/bmlt.wszf.org/public_html/yap/app/Http/Controllers/LegacyController.php(36): Illuminate\\Routing\\ResponseFactory->view('legacy', Array)
#18 /home/customer/www/bmlt.wszf.org/public_html/yap/app/Http/Controllers/LegacyController.php(18): App\\Http\\Controllers\\LegacyController->renderPath('/home/customer/...', 'text/xml')
#19 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\\Http\\Controllers\\LegacyController->index(Object(Illuminate\\Http\\Request), 'meeting-search....')
#20 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction('index', Array)
#21 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Route.php(261): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(App\\Http\\Controllers\\LegacyController), 'index')
#22 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Route.php(204): Illuminate\\Routing\\Route->runController()
#23 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(695): Illuminate\\Routing\\Route->run()
#24 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#25 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#26 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#27 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#28 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#29 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#30 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#31 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#32 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest(Object(Illuminate\\Http\\Request), Object(Illuminate\\Session\\Store), Object(Closure))
#33 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Session\\Middleware\\StartSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#34 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#35 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#36 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#37 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#38 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#39 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(697): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#40 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(672): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))
#41 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(636): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))
#42 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Routing/Router.php(625): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))
#43 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(166): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))
#44 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))
#45 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#46 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#47 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#48 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#49 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#50 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#51 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#52 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#53 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#54 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#55 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/fruitcake/laravel-cors/src/HandleCors.php(38): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#56 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fruitcake\\Cors\\HandleCors->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#57 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/fideloper/proxy/src/TrustProxies.php(57): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#58 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fideloper\\Proxy\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#59 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#60 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(141): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#61 /home/customer/www/bmlt.wszf.org/public_html/yap/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(110): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#62 /home/customer/www/bmlt.wszf.org/public_html/yap/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#63 {main}
"} 


```

**Comment by dgershman** [comment]
This error occurs when you have `mobile_check` enabled, when determining if to send SMS.  It's likely that this issue was there prior, but only after `mobile_check` was enabled on your instance.  I can some additional error catching here.  The number `266696687` literally means ANONYMOUS.

**Comment by dgershman** [comment]
Actually, this is unrelated to mobile check and appears to be an issue of sending SMS to Anonymous numbers.  This is not allowed by Twilio.  I'm not sure how this was not caught before.  There will be a fix for this in Yap 4.0.2.

**Comment by dgershman** [comment]
fix will be yap 4.0.2
